index.src.html

```diff
@@ -1201,6 +1213,8 @@
   required unsigned long cropHeight;
   unsigned long displayWidth;
   unsigned long displayHeight;
+
+  AvcEncoderConfig avc;
```

Comment:
Perhaps it could be named as `codecSpecificConfig`, and the type could be `(AvcEncoderConfig or CodecnameEncoderConfig)` in the future.